### PR TITLE
Basic refactoring of code

### PR DESCRIPTION
**Pull request by avikumar15**
Distributed cli.py into smaller python class files.

### CODE REVIEW

**Comment by narutosasuke30** [comment]
Looks really good. Some of suggestions I have:
Instead of the current file tree how about we change it to:
```text
src/cpcli
|-- __init__.py
|-- cmdline.py
|-- question.py
|-- scrapper.py
|-- testcase.py
|-- commands
|   |-- __init__.py
|   |-- download.py
|   |-- init.py
|   |-- run.py
|   |-- show.py
|   `-- testcase.py
|-- platforms
|   |-- __init__.py
|   |-- codechef.py
|   `-- codeforces.py
`-- utils
    |-- __init__.py
    |-- cmdtypes.py
    |-- config.py
    |-- constants.py
    |-- exceptions.py
    |-- misc.py
    `-- python.py
```

Changes done :
```diff
deleted:    src/cpcli/helpers/__init__.py
renamed:    src/cpcli/helpers/question.py -> src/cpcli/question.py
renamed:    src/cpcli/helpers/scrapper.py -> src/cpcli/scrapper.py
renamed:    src/cpcli/helpers/testcase.py -> src/cpcli/testcase.py
renamed:    src/cpcli/helpers/invalidcontesturi.py -> src/cpcli/utils/exceptions.py
```

File: src/cpcli/question.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import string
+import math
+import os
+from typing import Dict, List, Optional
+
+from cpcli.testcase import TestCase
+
+WHITE_SPACES = string.whitespace
```
Comment (narutosasuke30) [inline src/cpcli/question.py]: Move `WHITE_SPACES` to `utils/constants.py`. (Same for `testcase.py`)

**Review by narutosasuke30 — COMMENTED** [review]
Looks good. How about we change `Scrapper` to `Runner` ?

File: src/cpcli/question.py (hunk 1 above)
Comment (avikumar15) [reply src/cpcli/question.py]: Sure!

File: src/cpcli/commands/testcase.py [hunk 2]
```diff
@@ -31,7 +31,7 @@ def add_options(self, parser: ArgumentParser) -> None:
             help='Add a new custom test case'
         )
 
-    def run(self, args: Namespace, scraper: Scraper) -> None:
+    def run(self, args: Namespace, scraper: Runner) -> None:
```
Comment (narutosasuke30) [inline src/cpcli/commands/testcase.py]:
```suggestion
    def run(self, args: Namespace, runner: Runner) -> None:
```

This should be renamed as well? (same for all the other occurences)

**Review by narutosasuke30 — CHANGES_REQUESTED** [review]

**Review by narutosasuke30 — APPROVED** [review]

**Pull request merged**